language_names/manual.py

```diff
@@ -23,13 +24,13 @@ def populate_database():
     c = conn.cursor()
     c.execute('PRAGMA foreign_keys=OFF;')
     c.execute('BEGIN TRANSACTION;')
-    insert_values(c, 'language_names/fixes.tsv', 'fixes')
-    insert_values(c, 'language_names/additions.tsv', 'additions')
-    insert_values(c, 'language_names/turkic_fixes.tsv', 'fixes')
-    insert_values(c, 'language_names/turkic_langNames.tsv', 'languageNames')
-    insert_values(c, 'language_names/scraped.tsv', 'languageNames')
-    insert_values(c, 'language_names/scraped-sil.tsv', 'languageNames')
-    insert_values(c, 'language_names/variants.tsv', 'languageNames')
+    insert_values(c, sys.argv[1], 'fixes')
```

Comment:
There's no reason to make this hardcoded... just use a loop.

Reply:
How would I change second arguement then?

Reply:
I thought about using a loop but problem is different files do insertion in different databases ( fixes, additions, languageNames)

Reply:
In that case, I suggest just not passing in any arguments to the program and not using them either.

Reply:
okay I have changed the files and now I am not passing any arguements